frontend/src/lang/modules/en.ts

```diff
@@ -2376,7 +2376,7 @@ const message = {
         runtime: 'Runtime',
         image: 'Image',
         workDir: 'Working directory',
-        create: 'Create',
+        create: 'Create runtime',
         resource: 'Source',
         appstore: 'App Store',
         local: 'Local',
```

Comment:
There are no significant technical issues in the provided code differences. However, there are some minor style improvements that can be suggested:

1. **Consistent Formatting**: Ensure consistent spacing around operators (e.g., `:`) and within strings.

2. **Readability**: The line "localIP: 'Local IP'," could benefit from adding a dash after "IP" for better readability.

Here is an example of how it might look after applying these suggestions:

```javascript
const message = {
    host: 'Host | Hosts',
    system: 'System',
    security: 'Security',
    files: 'Files',
    monitor: 'Monitoring',
    terminal: 'Terminal | Terminals',
    settings: 'Settings | Settings',

    // Other keys...

    localIP: 'Local-IP',  // Consistent spacing and add a dash at the end

    // Continued lines...
}
```

These changes should improve the overall readability and consistency of the codebase. If you have more specific areas where you desire further optimization or adjustments, please let me know!